templates/gin/files/server.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	_ "github.com/go-sql-driver/mysql"
+	psh "github.com/platformsh/config-reader-go/v2"
+	sqldsn "github.com/platformsh/config-reader-go/v2/sqldsn"
+	"net/http"
+)
+
+var db = make(map[string]string)
```

Comment:
Why is this a global?

And... it's a string->string hash map?  I don't get it...